library/card/card.css

```diff
@@ -46,8 +46,7 @@
 
 }
 
-.dui-card__button {
-}
+.dui-card__button {}
```

Comment:
can be removed it not in use.

Reply:
No, that's a placeholder for future use.